ipatests/test_integration/test_ipa_cert_fix.py

```diff
@@ -228,11 +237,20 @@ def test_missing_startup(self, expire_cert_critical):
         result = self.master.run_command(['ipa-cert-fix', '-v'],
                                          stdin_text='yes\n',
                                          raiseonerr=False)
+
         err_msg1 = "ERROR: 'selftests.container.order.startup'"
         # check that pki-server cert-fix command fails
         err_msg2 = ("ERROR: CalledProcessError(Command "
                     "['pki-server', 'cert-fix'")
-        assert err_msg1 and err_msg2 in result.stderr_text
+        warn_msg = ("WARNING: No selftests configured in "
+                    f"{paths.CA_CS_CFG_PATH} "
+                    "(selftests.container.order.startup)")
+
+        if (tasks.get_pki_version(self.master)
+           < tasks.parse_version('10.11.0')):
+            assert err_msg1 and err_msg2 in result.stderr_text
```

Comment:
The above assert is equivalent to assert (err_msg1 is not None) and (err_msg2 in stderr), which is not what the test should check.